app/messages/send-notification.js

```diff
@@ -1,25 +1,25 @@
 import crypto from 'crypto'
-import { NotifyClient } from 'notifications-node-client'
-import { notifyConfig } from '../config/index.js'
+import notifyClient from '../clients/notify-client.js'
+import { saveToDatabase } from './save-to-database.js'
 
 const sendNotification = async (message) => {
-  const notifyClient = new NotifyClient(notifyConfig.get('notifyApiKey'))
-
   const emailAddresses = Array.isArray(message.body.data.commsAddress)
     ? message.body.data.commsAddress
     : [message.body.data.commsAddress]
 
   for (const emailAddress of emailAddresses) {
     try {
-      await notifyClient.sendEmail(
+      const response = await notifyClient.sendEmail(
         message.body.data.notifyTemplateId,
         emailAddress, {
           personalisation: message.body.data.personalisation,
           reference: crypto.randomUUID()
         }
       )
+      await saveToDatabase(message, response, null)
```

Comment:
This can be replaced to a direct call to `addSuccessLog` if repo mentioned in https://github.com/DEFRA/fcp-fd-comms/pull/3/files#r1836155456 is created.